infra/modules/azure_naming_convention/README.md

```diff
@@ -1,5 +1,199 @@
 # DX - Common Variables Validation
 
+This module generates output values that follow the naming conventions used in DX. It takes the following inputs:
+
+- `prefix`: The prefix for the naming convention (e.g., io, dx, ...).
+- `env_short`: A short identifier for the environment (e.g., “d”, "p", "u").
+- `location`: The Azure region or location (e.g. italynorth, westeurope, ...).
+- `domain`: (Optional) The domain name to include in the naming convention.
```

Comment:
I'll try to clarify what 'domain' means, same for 'app_name'

Reply:
Here, I’m not entirely sure what the domain actually represents; it’s been used in the organization for a long time, but its meaning has never been very clear.
I think the “app name” is self-explanatory, it’s simply the resource’s identifier. However, if you have suggestions for improving it, I’m open to ideas.

In any case, these are values already present across all repositories and modules, so changing their meaning could result in a significant update, which isn’t the main focus of this PR

Reply:
I'd add a description for app_name that make constraints explicit, ie in case there's a limit on the length. Moreover I'd add in the description that is the "resource identifier".